### PR TITLE
Show message when there are no sources for a page (no files to debug)

### PR DESCRIPTION
**Pull request by ktamir**
Associated Issues: #2371 #2231 

### Summary of Changes
When a page doesn't have sources, show an indicative message "This page has no sources". Right now nothing is shown.

### Test Plan
- Go to https://devtools-html.github.io/debugger-examples/
- Attach debugger to that tab
- Check if there is a message "This page has no sources."
- Go to a page with sources
- Check that the tree loads normally

### Screenshots/Videos (OPTIONAL)

debugger-html-no-sources-message

### CODE REVIEW

File: src/components/SourcesTree.js [hunk 1]
```diff
@@ -238,6 +238,13 @@ class SourcesTree extends Component {
       renderItem: this.renderItem
     });
 
+    const noSourcesMessage = dom.div({
+      className: "no-sources-message"
+    }, "This page has no sources.");
```
Comment (jasonLaster) [inline src/components/SourcesTree.js]: We should translate this with `L10N`

**Comment by wldcordeiro** [comment]
Correct me if I'm wrong but shouldn't we always have one source? The document itself? Or should we only show the document if we detect script tags in it? I'm confused. It seems like the bug is that in certain cases we aren't including the document as a source.

**Comment by ktamir** [comment]
@jasonLaster I committed the fix :)

**Comment by jasonLaster** [comment]
Thanks @ktamir. Unfortunately, we still need to update the properties file as well [l10n](https://github.com/devtools-html/debugger.html/blob/master/docs/local-development.md#l10n)

**Comment by jasonLaster** [comment]
Thanks Tamir!

Have you joined our slack? https://devtools-html-slack.herokuapp.com

**Pull request merged**